content/post/2017-04-29-introduction-of-liechi-s-site.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Introduction of Liechi's site
+author: Liechi Zhang
+date: '2017-04-29'
+slug: introduction-of-liechi-s-site
+categories: []
+tags:
+  - blogdown
+  - Hugo
+  - Github
+  - Netlify
+  - rstudio
+---
+
+Liechi's site is generated by [Hugo](https://gohugo.io/) through [blogdown](https://bookdown.org/yihui/blogdown/). The files used to build the site are hosted in a repository on [my github](https://github.com/liechi), and published to webpages by [netlify](https://www.netlify.com/). My posts are written with [Rstudio](https://www.rstudio.com/), in which the addins make it easy and natural to start writing. This site is an adaption of [Yihui's site](https://yihui.name/), that is to say, if you are interested in how it is built, you can refer to the excellent [introduction by Yihui](https://support.rbind.io/2017/04/25/yihui-website/) about the technical details.
+
+In my work, I spend lots of time trying to understand how plants pattern their cells, especially the patterning of stem cells and non-stem cells. I generate transgenic lines (or GMO, to frighten a few), perform live imaging (most of which are time-lapse), as well as a variety of molecular biology and biochemistry methods to achive my goal. I am also interested in the DNA, RNA sequecing techniques and the data analysis skills.
```

Comment:
That's nice to know I can include pictures to describe my work. I may upload later, I need some time to find an easy understanding and representative one.